src/transform.ts

```diff
@@ -105,3 +110,26 @@ document.body.appendChild(style);
 
 module.exports = exportedTokens`;
 }
+
+function processSass(src: string): string {
+  let sassLoadPaths: string;
+  try {
+    sassLoadPaths = fs
+      .readFileSync(path.join(CACHE_FOLDER, SASS_LOAD_PATHS_CONFIG), 'utf8')
+      .trim();
+  } catch {
+    // File doesn't exist
+    sassLoadPaths = JSON.stringify([]);
+  }
```

Comment:
Can we guarantee the error is always File doesn't exist?
Can we assert if the error is exactly that case?

Reply:
I think we can move to use `fs.existsSync` to check of the config file exists